web/pandas_web.py

```diff
@@ -465,9 +465,29 @@ def main(
             with open(os.path.join(source_path, fname), encoding="utf-8") as f:
                 content = f.read()
             if extension == ".md":
-                body = markdown.markdown(
-                    content, extensions=context["main"]["markdown_extensions"]
-                )
+                if "pdeps/" in fname:
```

Comment:
Because of the use of `TocExtension` below, we can't use the usual yaml. Not sure if there is a work around or a better approach here.